src/components/formItem/formItem.svelte

```diff
@@ -161,12 +163,17 @@
     justify-content: stretch;
     font: var(--font);
     gap: var(--gap);
-    transition:
-      color 0.2s ease-in-out;
+    transition: color 0.2s ease-in-out;
 
     .content {
+      flex: 1;
+
+      display: flex;
+      flex-direction: column;
+      justify-content: stretch;
       padding: 0 var(--leo-spacing-s);
-      margin-inline-end: auto;
+
+      cursor: text;
     }
```

Comment:
can we not just set `width: 100%` on the children of this element, rather than making it a flex?

Reply:
We probably could. I basically just copied what we had over in input.